test/langtools/jdk/javadoc/doclet/testSingletonLists/TestSingletonLists.java

```diff
@@ -270,8 +270,9 @@ public void startElement(String name, Map<String,String> attrs, boolean selfClos
             switch (name) {
                 case "ul": case "ol": case "dl":
                     counts.push(new TreeMap<>());
-                    if ("tag-list".equals(attrs.get("class"))) {
-                        inSeeList = true;
+                    String classAttr = attrs.get("class");
+                    if ("tag-list".equals(classAttr) || "toc-list".equals(classAttr) || "sub-nav-list".equals(classAttr)) {
```

Comment:
This is not wrong, but it might be a good place to use `Set.contains` in some manner.